pyproject.toml

```
@@ -0,0 +1,25 @@
+[project]
+name = "gait"
+dynamic = ["version"]
+authors = [
+  { name="Matt Grant", email="mattgrant33@gmail.com" },
```

Way to leave me out guyyyy 🔪 
```suggestion
  { name="Matt Grant", email="mattgrant33@gmail.com" },
  { name="Brandon Powers", email="brandonkpowers@gmail.com" },
```